docs/apis/_index.md

```diff
@@ -6,4 +6,48 @@ geekdocFilePath: _index.md
 geekdocCollapseSection: true
 ---
 
+{{< toc-tree >}}
+
 Infinite Scale provides a large set of different **application programming interfaces (APIs)**. Infinite Scale is built by microservices. That means many calls to "functions" in the code are remote calls.
+
+Basically we have two different API "universes": [HTTP](http) and [gRPC](grpc_apis).
+
+{{< columns >}} <!-- begin columns block -->
+
+{{< figure src="/ocis/static/http-logo.png" width="70%" alt="Image sourced from https://commons.wikimedia.org/ free of licenses" >}}
+<--->
+
+{{< figure src="/ocis/static/grpc-logo.png" width="70%" alt="Image sourced from https://grpc.io/ under CC 4.0 BY license" >}}
+
+{{< /columns >}}
+
+
+For inter-service-communication we are using mostly gRPC calls because it has some advantages. In the future, it may occur that clients use gRPC directly to make use of these advantages.
```

Comment:
```suggestion
For inter-service-communication we are using mostly gRPC calls because it has some advantages. In the future, clients may decide to use gRPC directly to make use of these advantages.
```